Makefile.am

```diff
@@ -14,9 +14,12 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
-SUBDIRS                 = doc
+## gnulib-tool says to mention this:
+ACLOCAL_AMFLAGS   = -I build-aux
+ 
+SUBDIRS           = doc build-aux test
 
-dist_doc_DATA           = AUTHORS COPYING COPYING.EXCEPTION README
+dist_doc_DATA           = AUTHORS COPYING COPYING.EXCEPTION README.md
```

Comment:
We need a file called README in the release tar all or Automake will complain. Can you please revert this change?

Reply:
the plain-text `README` already gets included by default without needing to be specified explicitly because it's one of the files that automake puts under its `am__DIST_COMMON` variable. If I change `README.md` to just `README`, then `make distcheck` fails with:
```
Making all in test
/Applications/Xcode.app/Contents/Developer/usr/bin/make  all-am
make[3]: Nothing to be done for `all-am'.
make[2]: *** No rule to make target `../../README.md', needed by `../../README'.  Stop.
make[1]: *** [all-recursive] Error 1
make: *** [distcheck] Error 1
```
If I change it back to `README.md`, however, it gets past that step successfully (it then proceeds to fail with an error from `texi2dvi` making `autoconf-archive.dvi` in `doc`, but that's probably due to my local TeX setup, and unrelated to the issue of what the README is named)